### PR TITLE
Update releases.rst

### PR DESCRIPTION
**Pull request by noniagriconomie**
Bump up the version numbers

### CODE REVIEW

File: contributing/community/releases.rst [hunk 1]
```diff
@@ -7,12 +7,12 @@ release and maintain its different versions.
 Symfony releases follow the `semantic versioning`_ strategy and they are
 published through a *time-based model*:
 
-* A new **Symfony patch version** (e.g. 2.8.15, 4.1.7) comes out roughly every
+* A new **Symfony patch version** (e.g. 4.4.9, 5.0.9, 5.1.0) comes out roughly every
```
Comment (wouterj) [inline contributing/community/releases.rst:10]: While 5.1.0 strictly is a patch version, I wouldn't list it here (it's a minor release)

Comment (noniagriconomie) [reply contributing/community/releases.rst:10]: should we wait 5.1.1 to be released? as this PR target 5.1, i wanted to put a 5.1 patch version

**Comment by javiereguiluz** [comment]
@noniagriconomie we've merged #13741, so we need to rebase this one. Thanks!

File: contributing/community/releases.rst (hunk 1 above)
Comment (javiereguiluz) [reply contributing/community/releases.rst:10]: Let's use `5.1.1` ... it doesn't exist yet, but it will exist very soon, so it's OK.

Comment (noniagriconomie) [reply contributing/community/releases.rst:10]: ok

**Comment by noniagriconomie** [comment]
I will do it soon ok 👍

**Comment by noniagriconomie** [comment]
@javiereguiluz done, i also just fix an url (tell me if you prefer to do it on another PR)

**Comment by javiereguiluz** [comment]
Antoine thanks a lot for rebasing this! Thanks for fixing the outdated link too 🙏

**Pull request merged**